util/uvmdvgen/uvmdvgen.py

```diff
@@ -120,6 +125,13 @@ def main():
         of the FuesSoC core files.""")
 
     args = parser.parse_args()
+
+    # The name should have no slashes, just a block name.
```

Comment:
Checking that it doesn't contain a forward slash feels oddly specific.  
IMO a better check would be to ensure its strictly alphanumeric plus `_`, i.e. `[a-zA-Z0-9_]`.